File: Core/CMS.Domain/Entities/Notification.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CMS.Domain.Entities
{
    public class Notification
    {
        [Key]
        public int ValueId { get; set; }
        public string EmployeeCode { get; set; }
        public string NotficationSubject { get; set; }
        public string NotficationMessage { get; set; }
        public DateTime NotificationDate { get; set; } = DateTime.Now;
        public bool isRead { get; set; } = false;
        public bool isDeleted { get; set; } = false;

        [NotMapped]
        public int totalRecords { get; set; }
    }
}

File: Frontend/CMS/src/app/models/notification.ts
export class Notification {
    valueId: number;
    employeeCode: string;
    notficationSubject: string;
    notficationMessage: string;
    notificationDate: string;
    totalRecords : number;
    isRead?: boolean;
    constructor(valueId:number, employeeCode:string, notficationSubject:string, notficationMessage:string,totalRecords : number, notficationDate:string)
    {
        this.valueId = valueId;
        this.employeeCode = employeeCode;
        this.notficationSubject = notficationSubject
        this.notficationMessage = notficationMessage;
        this.notificationDate = notficationDate;
        this.totalRecords = totalRecords;
    }
}
